src/bot.js

```diff
@@ -87,6 +86,56 @@ let initializeBot = () => {
 				logger.info('tft command...');
 				message.channel.send(('https://tracker.gg/tft/profile/riot/NA/'+data+'/overview'));
 			}
+
+
+
+			if (command === 'store' && data) {		
+				logger.info('store command...');
+
+				const StoreCommand = splitMessage[2];
+				logger.info(StoreCommand);
+
+				//creates empty string to be added to
+				let RetrieveCommand = '';
+
+				//adds to the empty string each string after the command used for calling
+				var i;
+				for(i = 3; i < splitMessage.length; i++){
+					RetrieveCommand += splitMessage[i] + ' ';
+				}
```

Comment:
replace the for loop with build-in slicing and joining function?
```
RetrieveCommand = splitMessage.slice(3) 
#extract the array from the 4th element to the last element, doesnt modify the original array
RetrieveCommand = RetrieveCommand.join(' ') #join an array of list with seperater ' ' 
```

Reply:
Ooo that's a great suggestion - could simplify it further too: 
` RetrieveCommand = splitMessage.slice(3).join(' ')`

Reply:
So I just replace
`RetrieveCommand = "";
var i;
for(i = 3; i < splitMessage.length; i++){
					RetrieveCommand += splitMessage[i] + ' ';
				}`
With `const RetrieveCommand = splitMessage.slice(3).join(' ')` ?

Reply:
ahh it didnt do new lines in here, i'll need to figure out formatting on github comment

Reply:
Can also remove the initialization of RetrieveCommand above, since you are initializing it with the slice/join here.

Reply:
Okay I believe I did that.

Reply:
use \`\`\` \`\`\` to enclose ur code if u got new line